addons/website_blog/views/snippets/snippets.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="website_blog" inherit_id="website.snippets" name="Snippet Blog">
+        <xpath expr="//t[@id='blog_latest_posts_list_hook']" position="replace">
+            <t t-snippet="website_blog.s_latest_posts" t-thumbnail="/website_blog/static/src/img/s_latest_posts.jpg"/>
+        </xpath>
+    </template>
```

Comment:
Since you are creating the file, do not indent the first level and add blank lines around the template